diff --git a/deps.edn b/deps.edn
index 67e3128c77..c656572f23 100644
--- a/deps.edn
+++ b/deps.edn
@@ -430,7 +430,9 @@
     "modules/drivers/sqlite/test"
     "modules/drivers/sqlserver/test"
     "modules/drivers/starburst/test"
-    "modules/drivers/vertica/test"]}
+    "modules/drivers/vertica/test"
+    "modules/drivers/duckdb/test"
+    ]}
 
 ;;; Linters

diff --git a/.clj-kondo/config.edn b/.clj-kondo/config.edn
index 85866f8d00..c494f0f34b 100644
--- a/.clj-kondo/config.edn
+++ b/.clj-kondo/config.edn
@@ -220,4 +220,5 @@
               :sqlite
               :sqlserver
               :starburst
-              :vertica}}
+              :vertica
+              :duckdb}}
 
